interp/notation.ml

```diff
@@ -98,21 +98,40 @@ let init_scope_map () =
 (**********************************************************************)
 (* Operations on scopes *)
 
+let warn_undeclared_scope =
+  CWarnings.create ~name:"undeclared-scope" ~category:"parsing"
```

Comment:
I suggest using `~category:"deprecated"` so that people can easily make all deprecated warnings fatal.

Reply:
OK, doing it, but maybe time to move to multi-category warnings.